<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadProcBoard</class>
 <widget class="QWidget" name="ThreadProcBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <property name="spacing">
    <number>15</number>
   </property>
   <item row="0" column="1">
    <widget class="QPushButton" name="btn_SetInitializingState">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string>Set Initializing State</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btn_SetStopState">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string>Set Stop State</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btn_SetRunningState">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string>Set Running State</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>89</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="GroupBox" name="gpb_InitialProc">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="GroupBox" name="gpb_AutoRunProc">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_Active">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string>Set Active</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="GroupBox" name="gpb_MonitorProc">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>CustomerWidget/GroupBox/groupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
